cmd/iam.go

```diff
@@ -1908,7 +1914,7 @@ func (sys *IAMSys) refreshEtcd() error {
 		return err
 	}
 	// load group policy mapping
-	if err := loadEtcdMappedPolicies(false, true, iamGroupPolicyMap); err != nil {
+	if err := loadEtcdMappedPolicies(false, false, iamGroupPolicyMap); err != nil {
```

Comment:
Why are you flipping the book here? This part looks incorrect.

Reply:
Because this will fail since groups don't exist on existing users.

Reply:
Also look at the same code in loadMappedPolicies() which is not using true as well.

Reply:
Pushed a correct fix here @kannappanr @harshavardhana